README.md

```diff
@@ -36,14 +37,15 @@ pub struct Person {
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn std::error::Error>> {
     /// Start by making a database connection.
-     let mut conn = ormlite::SqliteConnection::connect(":memory:").await.unwrap();
+    let mut conn = SqliteConnection::connect(":memory:").await.unwrap();
```

Comment:
Before, this was `ormlite::SqliteConnection` but really should have been `ormlite::sqlite::SqliteConnection`

Because the latter is pretty lengthy, I changed it to a `use` instead